src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -41,8 +51,10 @@ private BlockPlaceHandler onPlace() {
             @Override
             public void onPlayerPlace(BlockPlaceEvent e) {
                 Block b = e.getBlockPlaced();
-                BlockStorage.addBlockInfo(b, "text", "Edit me via the Projector");
-                BlockStorage.addBlockInfo(b, OFFSET_PARAMETER, "0.5");
+                BlockStorage.addBlockInfo(b, "text", ChatColors.color("Edit me via the Projector"));
```

Comment:
```suggestion
                BlockStorage.addBlockInfo(b, "text", "Edit me via the Projector");
```
Redundant call, the text currently has no colors.